src/rp2_common/pico_cyw43_arch/include/pico/cyw43_arch.h

```diff
@@ -194,13 +194,13 @@ int cyw43_arch_wifi_connect_blocking(const char *ssid, const char *pw, uint32_t
  * \ingroup pico_cyw43_arch
  *
  * \param ssid the network name to connect to
- * \param password the network password or NULL if there is no password required
+ * \param pw the network password or NULL if there is no password required
  * \param auth the authorization type to use when the password is enabled. Values are \ref CYW43_AUTH_WPA_TKIP_PSK,
  *             \ref CYW43_AUTH_WPA2_AES_PSK, or \ref CYW43_AUTH_WPA2_MIXED_PSK (see \ref CYW43_AUTH_)
  *
  * \return 0 if the initialization is successful, an error code otherwise \see pico_error_codes
  */
-int cyw43_arch_wifi_connect_timeout_ms(const char *ssid, const char *pw, uint32_t auth, uint32_t timeout);
+int cyw43_arch_wifi_connect_timeout_ms(const char *ssid, const char *pw, uint32_t auth, uint32_t timeout_ms);
```

Comment:
timeout_ms is missing from the documentation?

Reply:
Yup - I don't know the code well enough to add a description, but I can add the missing `\param timeout_ms` if you suggest something suitable.
(the actual change I'm making here is to rename the parameter in the `.h` to match the parameter-name in the `.c`)